### DISCUSSION TITLE
Integration of nominatim with an existing OSM server

### DISCUSSION THREAD

**Discussion by El977** [osm-search/Nominatim#2941] [Q&A] [answered]
I already have a working OSM Server configured How can I integrate Nominatim with an already existing OSM by making nominatum use an already existing database Thanks in advance

**Comment by mtmail** [comment, marked as answer]
Nominatim requires its own database. It uses different table structure, it imports different data.

**Reply by El977** [reply]
Thanks, then can I remove the old server and use Nominatim as a full OSM Server ? Or do you still need an old server?

**Reply by mtmail** [reply]
I assume by OSM server you mean a tile server (something that creates map tiles). Nominatim can't do that. You can install tile server and Nominatim software on the same server, but the database names have to be different (Nominatim's default name is `nominatim`).

**Reply by El977** [reply]
it's clear , thank you for information
